src/Symfony/Component/HttpFoundation/Tests/Session/Storage/NativeSessionStorageTest.php

```diff
@@ -198,17 +198,17 @@ public function testSetSaveHandler()
         $this->iniSet('session.save_handler', 'files');
         $storage = $this->getStorage();
         $storage->setSaveHandler();
-        $this->assertInstanceOf('Symfony\Component\HttpFoundation\Session\Storage\Proxy\SessionHandlerProxy', $storage->getSaveHandler());
+        $this->assertInstanceOf(SessionHandlerProxy::class, $storage->getSaveHandler());
```

Comment:
we don't change that to avoid conflicts unless the line needs to be touched anyway. so this should be reverted.